### ISSUE TITLE
[Continued] Closing modals should return keyboard focus to their originating button

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/sul-embed#1867]
[After consulting with the POs, I'm creating this issue again for future reference.] A compromise fix for this behavior was completed in #1736. This issue describes the ideal behavior, and technical reasons for the compromise fix.

### Ideal behavior

The user opens the Share&Download popover menu by clicking the Share&Download button. They make a selection from the menu, which opens either the Share or Download modal dialog. Upon closing the modal dialog, the Share&Download popover does not reopen. If the user is navigating by keyboard, then keyboard focus is returned to the Share&Download button—not the popover menu.

### Current behavior

After closing either the Share or Download modal dialog, the Share&Download popover menu reopens, and keyboard focus is redirected to the Share&Download popover.

### Background

The reason for reopening the popover menu is to improve keyboard accessibility (as described in #1736).
- The Share&Download popover menu is a dialog that opens another modal dialog (either Share or Download).
- According to dev: There seems to be an issue with browsers properly supporting the JavaScript focus action. When closing either the Share or Download modal dialog we're unable to get the focus set to the original Share&Download button... but if we reopen the Share&Download popover menu, the proper button there does have the focus.

### Steps to reproduce

Test keyboard focus. Use Tab/Shift+Tab and Return keys to navigate through the viewer.

**Comment by mjgiarlo** [comment]
I believe this should have been closed by #1999, closing
